### DISCUSSION TITLE
Remove Outdated Metadata

### DISCUSSION THREAD

**Discussion by Setherizor** [seaweedfs/seaweedfs#2547] [Q&A] [answered]
I am using seaweedfs with S3, and when reading through [the documentation](https://github.com/chrislusf/seaweedfs/wiki/Amazon-S3-API), I came across the sentence
> "A bucket can be deleted efficiently by deleting the whole collection.".

So, I tried it out by using `collection.delete` in the weed shell and was very confused when the filer still had metadata for the filed in the collection I just deleted.

Reading through [this issue](https://github.com/chrislusf/seaweedfs/issues/2421), and the documentation on `volume.fsck` brought me to using the command:
```
> volume.fsck -findMissingChunksInFiler
```
which worked wonderfully at identifying all the outdated metadata. Problem now, is how to clean out this metadata, is there something like `-reallyDeleteFromVolume` but that works to in reverse like `-findMissingChunksInFiler` to handle this?

**Comment by chrislusf** [comment, marked as answer]
you can just delete the folder.

To also delete the bucket metadata efficiently, use a filer store that supports https://github.com/chrislusf/seaweedfs/wiki/S3-API-FAQ#how-to-speed-up-bucket-deletion
